### ISSUE TITLE
[Feature Request] ElasticSearch support

### ISSUE THREAD

**Issue by cwap** [Vonage/Grafana_Status_panel#121]
Hi,

This plugin looks awesome. Sadly we're running Elastic Search in our production monitoring tools, so I can't  use it for my work projects. Would it be feasible and/or possible to add Elastic Search support?

**Comment by Atem18** [comment]
@cwap It works with ES. It's probably just not tested.

**Comment by Atem18** [comment]
Are you sure your elasticsearch data source is well configured ? Do you use it with others graphs ?

**Comment by cwap** [comment]
Yup, got plenty of other graphs up and running.

What I'm trying to achieve is a grid showing the current state of a bunch of Windows Services. I'm using metricbeat with the Windows Services metricset to poll the state of the services ( https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-metricset-windows-service.html ). Im confused about how I should query the data from Elastich Search. Should I do one query for each Windows service, or should I do a single query for all Windows Services and the group by service.displayname?

---- Aaaand as I'm writing this I get what I did wrong. I totally messed up the concept of this plugin :) It needs 1 graph for each cell in a the grid, right? So for example, for https://grafana.com/api/plugins/vonage-status-panel/versions/1.0.8/images/img/environment_snapshot.png you have added a panel for each server, correct?

So what I would want to do is add a panel for each Window service I want tracked and then set them up as a grid. Makes sense!

I think I got everything right now - and yeah, Elastic Search works just fine. I just needed to understand how to use it.

Thanks for the plugin!
